src/Compilers/Core/Portable/Emit/CommonPEModuleBuilder.cs

```diff
@@ -1088,6 +1095,7 @@ internal PrivateImplementationDetails GetPrivateImplClass(TSyntaxNode syntaxNode
                         this.GetSpecialType(SpecialType.System_Int16, syntaxNodeOpt, diagnostics),
                         this.GetSpecialType(SpecialType.System_Int32, syntaxNodeOpt, diagnostics),
                         this.GetSpecialType(SpecialType.System_Int64, syntaxNodeOpt, diagnostics),
+                        this.GetSpecialType(SpecialType.System_String, syntaxNodeOpt, diagnostics),
```

Comment:
>this.GetSpecialType(SpecialType.System_String, syntaxNodeOpt, diagnostics), [](http://example.com/codeflow?start=24&length=75)

Let's say we have a program that utilizes a private implementation detail class, but doesn't use string type, and the string type is not defined. Is this code going to break such a program?

Reply:
Please make sure we have a test for a scenario like that.